sintia/modules/user_stats.py

```diff
@@ -16,6 +16,16 @@
     )
 
 
+async def record_command(message: discord.Message, command: str) -> None:
+    await query_single_commit(
+        """
+        INSERT INTO command_activity_history (invoked_at, guild_id, channel_id, user_id, command)
```

Comment:
Are these values sanitized?

Reply:
Yes, they come from the library and since `aiomysql` uses prepared queries, there's no possibility of sql injection.